paasta_tools/contrib/create_paasta_playground.py

```diff
@@ -0,0 +1,57 @@
+import json
+import os
+
+from render_template import render_values
+
+
+def main():
+    cluster = os.environ.get(
+        "PAASTA_TEST_CLUSTER", "kind-{USER}-k8s-test".format(**os.environ)
```

Comment:
suggestion:
```suggestion
        "PAASTA_TEST_CLUSTER", "kind-{USER}-k8s-test".format(os.environ['USER'])
```
might be a clearer way of doing this